pkg/sql/schemachanger/scdeps/sctestdeps/backfill.go

```diff
@@ -127,3 +127,9 @@ func (s *indexSpanSplitter) MaybeSplitIndexSpans(
 ) error {
 	return nil
 }
+
+func (s *indexSpanSplitter) MaybeSplitIndexSpansForPartitioning(
+	_ context.Context, _ catalog.TableDescriptor, _ catalog.Index,
+) error {
+	return nil
+}
```

Comment:
move this while IndexSpanSplitter implementation to another source file than `backfill.go`.

Reply:
Done.